eZ/Publish/Core/Helper/Tests/PreviewLocationProviderTest.php

```diff
@@ -69,7 +67,7 @@ public function testGetPreviewLocationDraft()
 
         $location = $this->provider->loadMainLocation($contentId);
         $this->assertInstanceOf(APILocation::class, $location);
-        $this->assertSame($contentInfo, $location->contentInfo);
+        $this->assertSame($content, $location->content);
```

Comment:
`content` is protected, shouldn't we use `$location->getContent()` instead?

Reply:
On ValueObject protected means readOnly, it's handled by magic methods. If you want it to not be available it has to be private.

That said you are right for another reason, the property is not documented so it's technically not part of the API: https://github.com/ezsystems/ezpublish-kernel/blob/7.5/eZ/Publish/API/Repository/Values/Content/Location.php#L18-L31